projects/vdk-core/tests/vdk/internal/builtin_plugins/ingestion/test_ingester_base.py

```diff
@@ -41,7 +49,7 @@ def test_send_object_for_ingestion(mocked_send):
     destination_table = "a_destination_table"
     method = "test_method"
     target = "some_target"
-    ingester_base = create_ingester_base()
+    ingester_base = create_ingester_base(None)
```

Comment:
maybe def ```create_ingester_base(kwargs={})``` to simplify even more?

Reply:
Did something similar, as the IDE did not like `kwargs={}`.